modules/videoio/src/videoio_registry.cpp

```diff
@@ -333,7 +333,24 @@ class VideoBackendRegistry
 
 } // namespace
 
-namespace videoio_registry {
+namespace videoio_registry { namespace internal {
+    const VideoBackendInfo& getBackendInfo(VideoCaptureAPIs api)
+    {
+        if (api == CAP_ANY)
+            CV_Error(Error::StsError, "Wrong backend ID: CAP_ANY");
+
+        const int N = sizeof(builtin_backends)/sizeof(builtin_backends[0]);
```

Comment:
> builtin_backends

Not correct.
We should use `VideoBackendRegistry::enabledBackends` instead.
We need to add getter for this "protected" field:

```
const std::vector<VideoBackendInfo>& getEnabledBackends() const
{
    return enabledBackends;
}
```

Reply:
alright